appengine/README.txt

```diff
@@ -11,20 +11,21 @@ structure:
 
 blockly/
  |- app.yaml
+ |- deploy
  |- index.yaml
- |- index_redirect.py
+ |- main.py
  |- README.txt
+ |- requirements.txt
  |- storage.js
  |- storage.py
- |- closure-library/  (Optional)
  `- static/
      |- blocks/
      |- core/
      |- demos/
      |- generators/
      |- media/
      |- msg/
-     |- tests/
+     |- tests/  (Optional)
```

Comment:
Why is this optional?

Reply:
Historically we've indicated that the uncompiled code is optional for an App Engine deploy (not ours, but other people's).  Thus `closure-library/` and `blockly_uncompressed.js` were both labeled as "Optional".  Since the tests only run in uncompiled mode, they should be in that bucket too.  I guess `blocks/`, `core/` , and `generators/` should all be optional too.  Will add these.

Reply:
I prefer to keep the playground required.